app/faucet/management/commands/process_faucet_requests.py

```diff
@@ -0,0 +1,36 @@
+'''
+    Copyright (C) 2019 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+from django.core.management.base import BaseCommand
+
+from faucet.models import FaucetRequest
+from marketing.mails import reject_faucet_request
+
+
+class Command(BaseCommand):
+
+    help = 'processes easy to process faucet requests so that admins dont have to.'
+
+    def handle(self, *args, **options):
+        reject_comments = "Please tell us what you're planning on using these funds for in the comments section!  Thanks."
+        requests = FaucetRequest.objects.filter(rejected=False, fulfilled=False)
```

Comment:
Won't this function just reject all the unfulfilled and unrejected faucet requests? Do we wan't to filter on the comment being blank as you said in the PR description?

Reply:
67f05adc951a20b0e9a5040905482811a2972fe5